### ISSUE TITLE
wandb CLI (faulty import statement)

### ISSUE THREAD

**Issue by JPBeukes** [conda-forge/wandb-feedstock#17]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:
The wandb CLI breaks on Ubuntu 20.04 when installing with `conda install -c conda-forge wandb`.

Error Message (when calling `wandb --help`):
```
Traceback (most recent call last):
  File "~/local/miniconda3/envs/test/bin/wandb", line 10, in <module>
    sys.exit(cli())
TypeError: 'module' object is not callable
```

Fix (hack):  
Change the cli import statement in ~/local/miniconda3/envs/test/bin/wandb (line 6) from
```
from wandb.cli import cli
```
to
```
from wandb.cli.cli import cli
```
<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/jpb/local/miniconda3/envs/test:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
argh                      0.26.2          pyh9f0ad1d_1002    conda-forge
atomicwrites              1.3.0                      py_0    conda-forge
attrs                     19.3.0                     py_0    conda-forge
babel                     2.8.0                      py_0    conda-forge
bzip2                     1.0.8                h516909a_1    conda-forge
ca-certificates           2020.12.8            h06a4308_0  
certifi                   2020.12.5        py37h06a4308_0  
cffi                      1.13.2           py37h8022711_0    conda-forge
chardet                   3.0.4                 py37_1003    conda-forge
click                     7.1.2              pyh9f0ad1d_0    conda-forge
configparser              5.0.1                      py_0    conda-forge
cryptography              2.8              py37h72c5cf5_1    conda-forge
cudatoolkit               10.1.243             h6bb024c_0  
cycler                    0.10.0                     py_2    conda-forge
dbus                      1.13.6               he372182_0    conda-forge
docker-pycreds            0.4.0                      py_0    conda-forge
docutils                  0.16                     py37_0    conda-forge
expat                     2.2.5             he1b5a44_1004    conda-forge
fontconfig                2.13.1            h86ecdb6_1001    conda-forge
freetype                  2.10.0               he983fc9_1    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
gitdb                     4.0.5                      py_0    conda-forge
gitpython                 3.1.11                     py_0    conda-forge
glib                      2.58.3          py37h6f030ca_1002    conda-forge
gql                       0.1.0                      py_0    conda-forge
graphql-core              3.1.2                      py_0    conda-forge
gst-plugins-base          1.14.5               h0935bb2_1    conda-forge
gstreamer                 1.14.5               h36ae1b5_1    conda-forge
icu                       64.2                 he1b5a44_1    conda-forge
idna                      2.8                   py37_1000    conda-forge
imagesize                 1.2.0                      py_0    conda-forge
importlib_metadata        1.4.0                    py37_0    conda-forge
inflect                   4.0.0                    py37_1    conda-forge
intel-openmp              2019.4                      243  
jaraco.itertools          5.0.0                      py_0    conda-forge
jinja2                    2.10.3                     py_0    conda-forge
joblib                    0.14.0                     py_0    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
kiwisolver                1.1.0            py37hc9558a2_0    conda-forge
libblas                   3.8.0               14_openblas    conda-forge
libcblas                  3.8.0               14_openblas    conda-forge
libclang                  9.0.1           default_hde54327_0    conda-forge
libffi                    3.2.1             he1b5a44_1006    conda-forge
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_2    conda-forge
libiconv                  1.15              h516909a_1005    conda-forge
liblapack                 3.8.0               14_openblas    conda-forge
libllvm9                  9.0.1                hc9558a2_0    conda-forge
libopenblas               0.3.7                h6e990d7_2    conda-forge
libpng                    1.6.37               hed695b0_0    conda-forge
libprotobuf               3.13.0.1             h8b12597_0    conda-forge
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.1.0                hfc65ed5_0    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxkbcommon              0.9.1                hebb1f50_0    conda-forge
libxml2                   2.9.10               hee79883_0    conda-forge
lz4-c                     1.8.3             he1b5a44_1001    conda-forge
markupsafe                1.1.1            py37h516909a_0    conda-forge
matplotlib                3.1.2                    py37_0    conda-forge
matplotlib-base           3.1.2            py37h250f245_0    conda-forge
mkl                       2019.4                      243  
more-itertools            8.1.0                      py_0    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
ninja                     1.9.0                hc9558a2_1    conda-forge
nspr                      4.24                 he1b5a44_0    conda-forge
nss                       3.47                 he751ad9_0    conda-forge
numpy                     1.17.3           py37h95a1406_0    conda-forge
nvidia-ml                 7.352.0                    py_0    conda-forge
olefile                   0.46                       py_0    conda-forge
openssl                   1.1.1i               h27cfd23_0  
packaging                 20.1                       py_0    conda-forge
pathtools                 0.1.2                      py_1    conda-forge
pcre                      8.43                 he1b5a44_0    conda-forge
pillow                    6.2.1            py37h6b7be26_0    conda-forge
pip                       19.3.1                   py37_0    conda-forge
pluggy                    0.13.0                   py37_0    conda-forge
promise                   2.3              py37hc8dfbb8_2    conda-forge
protobuf                  3.13.0.1         py37h745909e_1    conda-forge
psutil                    5.7.3            py37hb5d75c8_0    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
py                        1.8.1                      py_0    conda-forge
pycparser                 2.19                     py37_1    conda-forge
pygments                  2.5.2                      py_0    conda-forge
pyopenssl                 19.1.0                   py37_0    conda-forge
pyparsing                 2.4.6                      py_0    conda-forge
pyqt                      5.12.3           py37hcca6a23_1    conda-forge
pyqt5-sip                 4.19.18                  pypi_0    pypi
pyqtwebengine             5.12.1                   pypi_0    pypi
pysocks                   1.7.1                    py37_0    conda-forge
pytest                    5.2.2                    py37_0    conda-forge
python                    3.7.3                h33d41f4_1    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python_abi                3.7                     1_cp37m    conda-forge
pytorch                   1.4.0           py3.7_cuda10.1.243_cudnn7.6.3_0    pytorch
pytz                      2019.3                     py_0    conda-forge
pyyaml                    5.3.1            py37hb5d75c8_1    conda-forge
qt                        5.12.5               hd8c4c69_1    conda-forge
readline                  8.0                  hf8c457e_0    conda-forge
requests                  2.22.0                   py37_1    conda-forge
scikit-learn              0.21.3           py37hcdab131_0    conda-forge
scipy                     1.3.1            py37h921218d_2    conda-forge
sentry-sdk                0.19.5             pyhd3deb0d_0    conda-forge
setuptools                41.6.0                   py37_1    conda-forge
shortuuid                 1.0.1            py37hc8dfbb8_3    conda-forge
six                       1.15.0           py37h06a4308_0  
smmap                     3.0.4              pyh9f0ad1d_0    conda-forge
snowballstemmer           2.0.0                      py_0    conda-forge
sphinx                    2.2.1                      py_0    conda-forge
sphinxcontrib-applehelp   1.0.1                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.1                      py_0    conda-forge
sphinxcontrib-htmlhelp    1.0.2                      py_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.2                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.3                      py_0    conda-forge
sqlite                    3.30.1               hcee41ef_0    conda-forge
tk                        8.6.9             hed695b0_1003    conda-forge
torchvision               0.5.0                py37_cu101    pytorch
tornado                   6.0.3            py37h516909a_0    conda-forge
urllib3                   1.25.7                   py37_0    conda-forge
wandb                     0.10.12            pyhd3deb0d_0    conda-forge
watchdog                  1.0.1            py37h89c1867_0    conda-forge
wcwidth                   0.1.8                      py_0    conda-forge
wheel                     0.33.6                   py37_0    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
yaml                      0.2.5                h516909a_0    conda-forge
zipp                      2.1.0                      py_0    conda-forge
zlib                      1.2.11            h516909a_1006    conda-forge
zstd                      1.4.3                h3b9ef0a_0    conda-forge


```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : test
    active env location : /home/jpb/local/miniconda3/envs/test
            shell level : 3
       user config file : /home/jpb/.condarc
 populated config files : 
          conda version : 4.8.4
    conda-build version : not installed
         python version : 3.7.4.final.0
       virtual packages : __cuda=10.1
                          __glibc=2.31
       base environment : /home/jpb/local/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/jpb/local/miniconda3/pkgs
                          /home/jpb/.conda/pkgs
       envs directories : /home/jpb/local/miniconda3/envs
                          /home/jpb/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.4 requests/2.23.0 CPython/3.7.4 Linux/5.4.0-58-generic ubuntu/20.04.1 glibc/2.31
                UID:GID : 1000:1000
             netrc file : /home/jpb/.netrc
           offline mode : False
```
</details>

**Comment by chrisburr** [comment]
This should be fixed by #21

**Comment by JPBeukes** [comment]
Thanks for your response. I noticed that it has fixed already been fixed in the latest version.
